jest.config.js

```diff
@@ -0,0 +1,185 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
I would suggest removing unnecessary comments.

Reply:
I'd agree. This file was created when jest --init was ran. I may keep it for now to see if there's anything I need to un-comment out later.